src/main/java/org/alliancegenome/curation_api/services/validation/dto/fms/GeneExpressionAnnotationFmsDTOValidator.java

```diff
@@ -46,17 +45,19 @@ public class GeneExpressionAnnotationFmsDTOValidator {
 	@Inject StageTermService stageTermService;
 	@Inject OntologyTermService ontologyTermService;
 
-	public GeneExpressionAnnotation validateAnnotation(GeneExpressionFmsDTO geneExpressionFmsDTO, BackendBulkDataProvider dataProvider) throws ValidationException {
+	public GeneExpressionAnnotation validateAnnotation(GeneExpressionFmsDTO geneExpressionFmsDTO, BackendBulkDataProvider dataProvider, Map<String, Set<String>> experiments) throws ValidationException {
 		ObjectResponse<GeneExpressionAnnotation> response = new ObjectResponse<>();
 		GeneExpressionAnnotation geneExpressionAnnotation;
+		String uniqueId;
+		String referenceCurie;
 
 		ObjectResponse<Reference> singleReferenceResponse = validateEvidence(geneExpressionFmsDTO);
 		if (singleReferenceResponse.hasErrors()) {
 			response.addErrorMessage("singleReference", singleReferenceResponse.errorMessagesString());
 			throw new ObjectValidationException(geneExpressionFmsDTO, response.errorMessagesString());
```

Comment:
Shouldn't be throwing the exception here as want to find any other validation errors before throwing exception

Reply:
To check if an annotation its already in the DB the uniqueId (containing AGR reference ID) must be used. At this point, if the publication is not found, the only option is throw the exception.

Reply:
You can just have a new GeneExpressionAnnotation object by default, and skip the search for the annotation if reference validation fails.

Reply:
Done